Vagrantfile

```diff
@@ -136,7 +131,7 @@ Vagrant.configure("2") do |config|
     build.vm.provision "ansible" do |ansible|
       ansible.playbook = "install_files/ansible-base/build-deb-pkgs.yml"
       ansible.verbose = 'v'
-      #ansible.skip_tags = [ "ossec" ]
+      ansible.skip_tags = ENV['BUILD_SKIP_TAGS']
```

Comment:
Skipping the ossec tags was disabled by default. @dolanjs, should this be left enabled?